assignments/week_1/exercise_1_2/exercise_1_2.cpp

```diff
@@ -0,0 +1,34 @@
+// Iterative C++ program to reverse an array and check if string is palindrome
+#include <stdio.h>
+#include <string.h>
+#include <iostream>
+
+int main()
+{
+    // Part 1: Array Reversal
+    int arr[] = {1, 2, 3, 4, 5, 6};
+    int n = sizeof(arr) / sizeof(arr[0]);
```

Comment:
In general, we should be lead by example with descriptive variable names. That said, in this case, I think we can just replace `n` with `arr.size()` when it's a vector.